scrapyd_k8s/launcher/k8s.py

```diff
@@ -23,6 +25,9 @@ def __init__(self, config):
         self._k8s = kubernetes.client.CoreV1Api()
         self._k8s_batch = kubernetes.client.BatchV1Api()
 
+    def get_node_name(self):
+        return "%s.%s" % (os.getenv('MY_DEPLOYMENT_NAME'), os.getenv('MY_POD_NAMESPACE'))
+
```

Comment:
Let's have a fallback here. Someone's deployment may _not_ have these variables set.

Reply:
What kind of fallback do you want here?

Reply:
Are there any variables that are set by default in Kubernetes pods that could be useful?
If not, then we could just use `default` here.

Reply:
Thanks for looking this up!
I'm doubting a bit whether we
1. keep it simple stupid (and fallback to something like `default`) - as you can always set the node name in the config
2. return the pod name, so that there is at least something (but returns a new pod name after each restart, which would be polluting systems showing a list of node names, where multiple scrapyd servers are being accessed)
3. remove the suffix from the pod name to return the deployment name - this is a bit of a hack, and may give weird results if someone does not use a deployment, for example

I think the first option is actually the simplest to understand, I think.
Any thoughts?